src/vivarium_nih_us_cvd/tools/make_artifacts.py

```diff
@@ -99,10 +99,8 @@ def build_artifacts(
     verbose
         How noisy the logger should be.
     """
-    import vivarium_cluster_tools as vct
-
     output_dir = Path(output_dir)
-    vct.mkdir(output_dir, parents=True, exists_ok=True)
+    mkdir(output_dir, parents=True, exists_ok=True)
```

Comment:
Why did you change this import?

Reply:
B/c this was importing vct in make artifacts (the "data" setup requirement")

Reply:
By using option 2, the `data` requirement includes `cluster`, so you maybe don't need to do this anymore?

Reply:
Oops, yeah good call. I do kinda feel like the `mkdir` utility shouldn't be in vct anyway and should just be a part of the model repos but won't pull that string for now